src/foremast/utils/get_sns_topic_arn.py

```diff
@@ -20,6 +20,8 @@ def get_sns_topic_arn(topic_name, account, region):
         str: ARN for requested topic name
 
     """
+    if topic_name.count(':') == 5:
```

Comment:
Can you maybe add an additional statement just to do some basic sanitization such as:
`if topic_name.count(':') == 5 and topic_name.contains('arn:aws:sns'):`

Reply:
we could go all out regex to be absolutely sure